<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>color_set</class>
 <widget class="QWidget" name="color_set">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>颜色设置</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>9</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>颜色</string>
   </property>
  </widget>
  <widget class="QLabel" name="color_edit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>29</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>最大可观测强度值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>最小可观测强度值</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="strength_spin">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>20000</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="mini_strength_spin">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="initial">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>JET</string>
   </property>
  </widget>
  <widget class="QPushButton" name="initial2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>绿色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="initial3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>黄色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="initial4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>蓝色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseColor">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选择颜色</string>
   </property>
  </widget>
  <widget class="QWidget" name="part1_colorbar" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="part1_strength" native="true">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>81</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
